src/foam/nanos/column_config/CSVTableOutputter.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.column',
```

Comment:
The file system path mush match the package.
please change the directory name to 'column' from column_config

Reply:
fixed